Priyaa/HTML Basics Remake/Assignment-3.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
```

Comment:
Missing head tag. Refer this: [HTML Head - w3schools.com](https://www.w3schools.com/html/html_head.asp)